Cargo.toml

```diff
@@ -19,6 +19,13 @@ hyper-rustls = { version = "0.22.1", optional = true }
 log = "0.4.6"
 pin-project = "1.0.5"
 tokio = { version = "1.2.0", features = ["time"] }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
With the SDK test harness, we opted to avoid this optional dependency process by favoring cargo workspaces instead. We should probably do that here as well.